### PR TITLE
Multilayer SVG gerber layers

### PR DESCRIPTION
**Pull request by anas-sarkez**
Now each layer will create output svg file

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -174,4 +174,7 @@ dist
 # Finder (MacOS) folder config
 .DS_Store
 
-.vscode
\ No newline at end of file
+.vscode
+# yalc
+.yalc
+yalc.lock
```
Comment (seveibar) [inline .gitignore:180]:
```suggestion
yalc.lock
package-lock.json
```

File: package-lock.json [hunk 2]
```diff
@@ -0,0 +1,3271 @@
+{
```
Comment (seveibar) [inline package-lock.json:1]: difficult to keep track of but this repo uses `bun` so technically no need for a `package-lock.json` (to be honest, I wish bun just used package lock but 🤷 )

File: tests/fixtures/__snapshots__/simple1bottom.snap.svg [hunk 3]
```diff
@@ -0,0 +1,7 @@
+<svg version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" stroke-linecap="round" stroke-linejoin="round" stroke-width="0" fill-rule="evenodd" clip-rule="evenodd" id="NDmzwLDbaDfQ_bottom" viewBox="0 -20000 20000 20000" width="20in" height="20in"><defs><style>.NDmzwLDbaDfQ_fr4 {color: #666666;}
```
Comment (seveibar) [inline tests/fixtures/__snapshots__/simple1bottom.snap.svg:1]: this just looks like a gray square  on github, did something go wrong?

File: tests/fixtures/preload.ts [hunk 4]
```diff
@@ -17,11 +17,14 @@ async function toMatchGerberSnapshot(
...
     // We'll use the top layer SVG for comparison, but you could choose bottom or both
-    const svg = stackup.top.svg
-
-    return expect(svg).toMatchSvgSnapshot(testPathOriginal, svgName)
+    for (const item of Object.keys(stackup)) {
+        if (stackup[item].svg) {
+          gerbersArray.push(expect(stackup[item].svg).toMatchSvgSnapshot(import.meta.path, svgName + `${item}`))
+        }
+    }
+      return gerbersArray;
```
Comment (seveibar) [inline tests/fixtures/preload.ts:27]:
```suggestion
    for (const layerName of Object.keys(stackup)) {
        if (stackup[layerName].svg) {
          gerbersArray.push(expect(stackup[item].svg).toMatchSvgSnapshot(import.meta.path, svgName + `-${layerName}`))
        }
    }
      return gerbersArray;
```

File: tests/generate-board-outline-gerber.test.ts [hunk 5]
```diff
@@ -6,7 +6,7 @@ import {
 } from "src/stringify-gerber"
 import { maybeOutputGerber } from "tests/fixtures/maybe-output-gerber"
 import gerberToSvg from "gerber-to-svg"
-
+import { toMatchGerberSnapshot } from "./fixtures/preload"
```
Comment (seveibar) [inline tests/generate-board-outline-gerber.test.ts:9]: you don't need to import this, it should be registered on the global expect object

File: tests/fixtures/preload.ts (hunk 4 above)
Comment (seveibar) [inline tests/fixtures/preload.ts:27]: this method has to return a `MatchOutput` or something for it to be compliant with bun, you'll need to combine them (types will be useful here)

**Review by seveibar — CHANGES_REQUESTED** [review]

File: tests/fixtures/__snapshots__/simple1bottom.snap.svg (hunk 3 above)
Comment (ShiboSoftwareDev) [reply tests/fixtures/__snapshots__/simple1bottom.snap.svg:1]: It has always been like that

**Pull request closed without merging**